tests/core/actions/test_forms.py

```diff
@@ -81,8 +81,8 @@ async def test_activate_and_immediate_deactivate():
     assert events == [
         Form(form_name),
         SlotSet(slot_name, slot_value),
-        Form(None),
         SlotSet(REQUESTED_SLOT, None),
+        Form(None),
```

Comment:
I think this order is even better, since it represents the fact that requested slot was set to None inside the form